wiremock/mappings/ApAndOAsys_GetRiskManagementPlan.json

```diff
@@ -0,0 +1,38 @@
+{
+  "id": "01794bde-9568-4795-b4d9-5e09105f3ba0",
+  "request": {
+    "method": "GET",
+    "urlPathPattern": "/risk-management-plan/(.*)"
+  },
+  "response": {
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "status": 200,
+    "jsonBody": {
+      "totalElements": 1,
+      "totalPages": 1,
+      "number": 1,
+      "content": {
```

Comment:
Should this be an array with one object in it, rather than an object as the other fields allude to pagination?

Reply:
yes, thanks for spotting this -- a case of unthinking copy/paste on my part!